openc3-init/plugins/packages/openc3-tool-tlmviewer/src/tools/TlmViewer/TlmViewer.vue

```diff
@@ -169,6 +171,9 @@ export default {
         }
         this.updateScreens()
       })
+    Api.get('/openc3-api/autocomplete/keywords/screen').then((response) => {
+      this.keywords = response.data
```

Comment:
What about autocomplete for target,packet,item in valueWidgets?

Reply:
Yeah I punted on that for now. Might be a way to incorporate the existing stuff in there.